src/__tests__/CompareResults/SubtestsRevisionRow.test.tsx

```diff
@@ -83,6 +83,9 @@ describe('SubtestsRevisionRow Component', () => {
       />,
     );
 
+    // wait until all elements render correctly
+    await screen.findByText('browser.html');
+
     expect(document.body).toMatchSnapshot();
```

Comment:
See the changes below, you'll see that the snapshots were empty before, because the component wasn't rendered.